stix_shifter/src/modules/splunk/splunk_query_constructor.py

```diff
@@ -117,7 +123,9 @@ class _ObservationExpressionTranslator:
         ComparisonComparators.In: encoders.set,
         ComparisonComparators.Matches: encoders.matches,
         ComparisonExpressionOperators.And: 'AND',
-        ComparisonExpressionOperators.Or: 'OR'
+        ComparisonExpressionOperators.Or: 'OR',
+        ComparisonComparators.IsSuperSet: "=", # Splunk cidrmatch('<ip in CIDR notation>', ip) and = operator give the same result
+        ComparisonComparators.IsSubSet: "=" # cidrmatch function can be used for both issuperset and issubset operator
```

Comment:
may be my lack of knowledge, how does superset and subset works with the same operator? I don't see any difference in the translation. the splunk query translation returns the same result in both cases.